jetson/percep/main.cpp

```diff
@@ -143,12 +142,12 @@ int main() {
 
         //Run Obstacle Detection
         pointcloud.pcl_obstacle_detection();  
-        obstacle_return obstacle_detection (pointcloud.bearing, pointcloud.distance);
+        obstacle_return obstacle_detection (pointcloud.leftBearing, pointcloud.rightBearing, pointcloud.distance);
```

Comment:
Could you rename obstacle_detection to be obstacleOutput? The current name is misleading